org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -126,6 +127,9 @@ public void start(BundleContext bundleContext) throws BackingStoreException {
 
 		// Set the ID to use for preference lookups
 		JavaManipulation.setPreferenceNodeId(PLUGIN_ID);
```

Comment:
I'd move JavaManipulation.setPreferenceNodeId(PLUGIN_ID); before instantiating the preferencesManager